KarhooSDK.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -0,0 +1,34 @@
+{
```

Comment:
Do we need to have this file as part of this PR?

Reply:
It was automatically added, just like cocoapods and its pod lock file worked whenever there was a pod install/update ran. Could just gitignore the file though